scripts/dts/edtlib.py

```diff
@@ -233,6 +229,7 @@ def _init_devices(self):
         for dev in self.devices:
             # These depend on all Device objects having been created, so we do
             # them separately
+            dev._init_props()  # Needs to be here due to 'type: phandle'
```

Comment:
Can you clarify the comment.  I assume we need to handle `phandle` first.  Can you add why.

Reply:
It's because `Property.val` actually becomes the pointed-to `Device` instance. For that to work, it must already have been created.

I'll tweak the comment.

Reply:
Done